int16 process                       # trạng thái xử lý hiện tại của AGV
int16 id_action
float64 x
float64 y
float64 angle 
float64 vel_linear_x
float64 vel_angular_z
bool is_reach_target_point           # đã chạm tới đích chưa ??